indicators/NO_AATS_001/R/NO_AATS_001.qmd

```diff
@@ -0,0 +1,1010 @@
+
+Uncertainties relate to the variability in the frequency of alien species among NFI plots and recording errors for tree species within NFI plots.
+There is also uncertainty related to the setting of the lower reference value and to the shape of the relationship between indicator values and ecosystem condition.
+
+### 5.2 Reported uncertainties
+<!-- Describe the nature of the errors you (hopefully) provide alongside the point estimate for the condition metric. It is especially important to clarify if the errors are inferential (referring to the uncertainty of the point estimate) or descriptive (referreing to the data itself). Here' an example text: "What we report as uncertainties in @tbl-finalTable are inferential and refer to the uncertainty around the point estimates. It does not directly describe variation in the data itself. 
+However, the variation in the mean is defined by the spatial variation only, and quartiles are obtained by bootstrapping the set of localities within a region, and calculating the indicator for each of 9999 such samples".   -->
+
+Uncertainty is quantified by bootstrapping.
+The uncertainties reported here are inferential and refer to the uncertainty around the point estimates, reflecting spatial variation among NFI plots and recording errors.
+Additional uncertainties related to the setting of the lower reference value and the shape of the relationship between indicator values and ecosystem condition are not quantified.
+
+
+## 6. References
+
+::: {#refs}
+:::
+
+## 7. Import and prepare datasets
+
+<!--# All the data import and preparation should be done in this chapter, before the analysis chapter. As a minimum, add code for importing data, and describe or show the data structure. You may also explain certain variables, what they represent, and their distributions -->
+
+### 7.1 Alien coniferous tree species
+
+All trees larger than 5 cm dbh are recorded and identified for each 250 m² NFI plot according to methods described in (@viken2021-feltinstruks).
+For this indicator, data for the NFI variables `introgran_pr` and `introfuru_pr` from the last 5 years including the relevant reporting year are used.
+Aggregated area-weighted data must be used for appropriate spatial units such as counties, regions or the whole country.
+As the NFI plots vary in density for forest below/above the coniferous forest line and in Finnmark, each type of plot represents a different part of all forest land.
+In addition, some NFI plots are divided in two parts in the data.
+These variations in plot representativity must be accounted for in aggregated estimates.
+Data from NFI contain an area-weighting variable to be used in aggregating NFI data for spatial units larger than individual NFI plots.
+
+```{r load-data}
+
+# Load main data file
+lsk_data_raw <- read_csv(here("indicators", "NO_AATS_001", "data", "LSK_data_naturregnskap.csv"), 
```

Comment:
Since we cant share the data, we need to make alle the code `eval: false` so that ecRxiv admins can render the html. If you save the results table as `data/resuts.rds` on you machine and read it back in the script, then we can at least show that.